evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -140,24 +140,38 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
             });
 
             slider.on("click", "[data-action]", event => {
-                handle($(event.target).data("action"), true);
+                press($(event.target).data("action"), !!event.originalEvent);
+            });
+            slider.on("click", "[data-slide]", event => {
+                slide($(event.target).data("slide"));
+                updateButtons();
+            });
+            slider.on("click", "[data-repeat]", event => {
+                repeat($(event.target).data("repeat"));
+                updateButtons();
             });
 
             $(document).keydown(event => {
                 let actions = {
-                    37: "slide-left",        // arrow left
-                    39: "slide-right",       // arrow right
-                    74: "publish",           // j
-                    75: "make_private",      // k
-                    76: "hide",              // l
-                     8: "unreview",          // backspace
-                    13: "next-course",       // enter
-                    77: "repeat-unreviewed", // m
-                    78: "repeat-all"         // n
+                    37: "[data-slide=left]",          // arrow left
+                    39: "[data-slide=right]",         // arrow right
+                    74: "[data-action=publish]",      // j
+                    75: "[data-action=make_private]", // k
+                    76: "[data-action=hide]",         // l
+                     8: ":not(.btn-outline-secondary)[data-action]", // backspace
+                    13: "[data-url=next-course]",     // enter
+                    77: "[data-repeat=unreviewed]",   // m
+                    78: "[data-repeat=all]"           // n
                 };
 
                 if(actions[event.which]) {
-                    handle(actions[event.which]);
+                    let element = slider.find(actions[event.which]);
+                    // review hotkeys have no toggle behaviour
+                    if(actions[event.which].startsWith("[data-action")) {
+                        element.click();
+                    } else if(element.length) {
+                        element[0].click();
+                    }
```

Comment:
Please ignore how the condition is expressed, I will find something better than a bracket which will remain open to you the rest of the day. In fact, `publish`, `make_private` and `hide` all take the upper branch and `unreview` with all other actions take the lower.

Reply:
Yes, I think that `.startsWith("[data=action")` is not really clear about what it is doing. I just want to hear a "yes, this approach is more sound" or "please revert to the old one, that looks not so awesome i imagined" (Yes, still not a fan of your mental model) After that, I will remove some other minor flaws in the code

Reply:
i'm not sure why the [0] is needed, but if it is necessary, can't you put a `:first` or something similar into the selector and remove the branching here?

Reply:
The `[0]` would be more equivalent to `.eq(0)`, wrapping the jQuery object into a DOM object, to fake a real click

Reply:
so, can you put a :first into the selector and remove the branching?

also, in the previous review i proposed to include in the selector that the button must be active for it to be selected, so that the hotkey for a pressed button actually does nothing and does not pass on a toggle variable which is later used to determine that nothing should be done. did you try that?